fmralign/alignment_methods.py

```diff
@@ -13,6 +11,13 @@
 from sklearn.base import BaseEstimator, TransformerMixin
 from sklearn.linear_model import RidgeCV
 from sklearn.metrics.pairwise import pairwise_distances
+from .hyperalignment.piecewise_alignment import PiecewiseAlignment
+from .hyperalignment.linalg import safe_svd, svd_pca
+
+import jax
```

Comment:
We don't want this here ; we only wanted to import jax if it's actually being called, since it's a heavier dependency. That's why it was nested inside the function (and why it's handled separately in the pyproject.toml)

Reply:
sorry, just modified that cause the code did not work for me, putting everything back ;)